packages/sign_in_with_apple/lib/src/exceptions.dart

```diff
@@ -15,39 +15,39 @@ abstract class SignInWithAppleException implements Exception {
     switch (exception.code) {
       case 'not-supported':
         return SignInWithAppleNotSupportedException(
-          message: exception.message,
+          message: exception.message ?? 'no message provided',
```

Comment:
Or we also make the message optional all the way through.

Reply:
Will keep the fallback for now, as the optional message seems to be a specialty of `PlatformException` only.